requirements.txt

```diff
@@ -36,14 +36,14 @@ pyOpenSSL==22.0.0
 python-dateutil==2.8.2
 PyYAML==6.0
 requests==2.27.1
-sentry-sdk==1.5.9
+sentry-sdk==1.5.10
 service-identity==21.1.0
 six==1.16.0
 SQLAlchemy==1.4.35
 treq==22.2.0
 Twisted==22.4.0
 txaio==22.2.1
-typing_extensions==4.1.1
+typing_extensions==4.2.0
```

Comment:
Why do you update requirements? If you need these versions, can you please do it in a separated PR?

Reply:
I'll revert the requirement updates, they shouldn't be needed.